policies/aws_cloudtrail_policies/aws_cloudtrail_enabled.py

```diff
@@ -1,12 +1,25 @@
 def policy(resource):
-    if not resource["Trails"]:
+    # pylint: disable=R1260
+    if not resource.get("Trails"):
         return False
 
-    if not resource["GlobalEventSelectors"]:
+    if not (resource.get("GlobalEventSelectors") or resource.get("GlobalAdvancedEventSelectors")):
         return False
 
-    for selector in resource["GlobalEventSelectors"]:
-        if selector["IncludeManagementEvents"] and selector["ReadWriteType"] == "All":
+    for selector in resource.get("GlobalEventSelectors", [{}]):
+        if selector.get("IncludeManagementEvents") and selector.get("ReadWriteType") == "All":
             return True
 
+    for advanced_selector in resource.get("GlobalAdvancedEventSelectors", [{}]):
+        management_present = False
+        readonly_excluded = True
```

Comment:
AWS Docs for this suggestion are linked in the comment below

```suggestion
        # if not explicitly set otherwise, the default state of readOnly is include
        # events where readOnly true and events where readOnly false
        readonly_excluded = False
```

Reply:
readonly_excluded is referring to the presence of the readOnly flag in the selector set.